Remotion/Web/Development.WebTesting/DownloadInfrastructure/DownloadedFileFinder.cs

```diff
@@ -55,7 +55,7 @@ public FileInformationTuple (DateTime lastWriteTimeUtc, long length)
     private readonly string _partialFileExtension;
     private readonly IDownloadFileFinderStrategy _downloadFileFinderStrategy;
     private readonly TimeSpan _downloadStartedGracePeriod;
-
+    private PartialFileState _partialFileStateOfCurrentDownload;

```

Comment:
Looking at the final code, I don't see a need to have this as a field. We can always pass the value as a parameter (there's just two methods that need this). 

Reasoning: 
* Before this change, DownloadedFileFinder does not have mutable state. It doesn't matter if we create a new one for each request or if there's no reasonable scenario where you would want to run two of them in parallel. It still keeps the state and the behavior of the type pure if we don't introduce mutable state. 
* When we do one day refactor this, the cut line will be along the state object (since it's already aptly named) and mutable state must then gravitate towards this type. Having the state as variables instead of fields on another type, keeps the state isoated.
* It feels like it's the right thing to do (TM).